### ISSUE TITLE
Popupmenu is behind taskbar

### ISSUE THREAD

**Issue by gcardi** [HemulGM/FMXTrayIcon#1]
Hi.

I tried to use your FMXTrayIcon component. It works well except for one minor flaw: the associated popup menu always appears behind the traybar icon window. 

(See image)

Thank you

[Image](https://drive.google.com/file/d/1Pkmv886Vamd1Rnko2WD00B--5BFJedG-/view?usp=sharing)

**Comment by gcardi** [comment]
Resolved by adding...

```
    ::SetWindowPos(
        Fmx::Platform::Win::ApplicationHWND(),
        HWND_TOPMOST, 0, 0, 0, 0,
        SWP_NOSIZE | SWP_NOMOVE | SWP_NOACTIVATE 
    );
```

after 

```
    trayIcon_->Show();
```

Example:

```
void TfrmMain::MinimizeToTrayBar()
{
    Hide();
    trayIcon_->Show();

    ::SetWindowPos(
        Fmx::Platform::Win::ApplicationHWND(),
        HWND_TOPMOST, 0, 0, 0, 0,
        SWP_NOSIZE | SWP_NOMOVE | SWP_NOACTIVATE 
    );

    HideAppOnTaskbar();
}
```

Regards

**Comment by HemulGM** [comment]
Oh, thank you very much. I have not made edits for a long time, and I myself have used it for a long time. But I think it's time to fix it. Thanks again. Maybe today I will fix it using your suggestion

**Comment by gcardi** [comment]
Hi.

I noticed that after minimizing the main window in the tray icon as per the code in the previous posts, when I would restore the window it would always remain topmost, always overstanding all other applications. Since this is an annoying problem, I solved it by declaring HWND_NOTOPMOST in the call to SetWindowPos when restoring from the tray bar (by double-clicking on the tray icon).

```Cpp
void TfrmMain::RestoreFromTrayBar()
{
    ::SetWindowPos(
        Fmx::Platform::Win::WindowHandleToPlatform( Handle )->Wnd,
        HWND_NOTOPMOST,
        0, 0, 0, 0,
        SWP_NOSIZE | SWP_NOMOVE | SWP_NOACTIVATE | SWP_SHOWWINDOW
    );
    trayIcon_->Hide();
    ShowAppOnTaskbar();
}
```

Note, however, that this time I did not use the application handle, but the main window handle. 

To recap: when minimizing you should use the application handle and the HWND_TOPMOST flag in the SetWindowPos, and when restoring you should use the main window handle and HWND_NOTOPMOST when calling the SetWindowsPos.

**Comment by HemulGM** [comment]
Fixed popup menu display. Do additional actions before or after - not necessary
